src/components/Tooltip/PopoverAnchorTooltip.js

```diff
@@ -0,0 +1,53 @@
+import React, {useContext, useRef, useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {propTypes as tooltipPropTypes, defaultProps as tooltipDefaultProps} from './tooltipPropTypes';
+import BaseTooltip from './BaseTooltip';
+import {PopoverContext} from '../PopoverProvider';
+
+const propTypes = {
+    ...tooltipPropTypes,
+
+    /** Whether the actual Tooltip should be rendered. If false, it's just going to return the children */
+    shouldRender: PropTypes.bool,
+};
+
+const defaultProps = {
+    ...tooltipDefaultProps,
+    shouldRender: true,
+};
+
+function PopoverAnchorTooltip({shouldRender, children, ...props}) {
+    const {isOpen, popover} = useContext(PopoverContext);
+
+    const tooltipRef = useRef(null);
+
+    const isPopoverRelatedToTooltipOpen = useMemo(() => {
+        // eslint-disable-next-line
+        const tooltipNode = tooltipRef.current ? tooltipRef.current._childNode : null;
+        if (isOpen && popover && popover.anchorRef.current && tooltipNode && (tooltipNode.contains(popover.anchorRef.current) || tooltipNode === popover.anchorRef.current)) {
```

Comment:
Is `popover.anchorRef` always defined? It seems like we have some [null checks](https://github.com/Expensify/App/blob/fe3e8bc3118816086bde2e28009a9018c41eba71/src/components/PopoverProvider/index.js#L41-L42) for it in PopoverContextProvider